src/TestFramework/MSTest.Core/Attributes/VSTestAttributes.cs

```diff
@@ -164,6 +195,20 @@ public sealed class ClassInitializeAttribute : Attribute
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
     public sealed class ClassCleanupAttribute : Attribute
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ClassCleanupAttribute"/> class.
+        /// ClassCleanupAttribute
+        /// </summary>
+        /// <param name="inheritanceBehavior">none</param>
+        public ClassCleanupAttribute(InheritanceBehavior inheritanceBehavior = InheritanceBehavior.None)
```

Comment:
A more friendly back compat change would be to add two ctors here - the default and the one that takes in InheritanceBehavior enum. What we currently have here would break binary compat.

Reply:
Done, added one more non-param ctor in both ClassInitialize and Cleanup attrs.